crates/ty_python_semantic/resources/mdtest/typed_dict.md

```diff
@@ -97,10 +97,18 @@ from typing import TypedDict
 from typing_extensions import Required, NotRequired
 
 Person = TypedDict("Person", {"name": Required[str], "age": int | None})
-
 reveal_type(Person)  # revealed: typing.TypedDict
 ```
 
+The `TypedDict` schema must be passed directly as the second argument:
+
+```py
+fields = {"name": str}
+
+# error: [invalid-argument-type] "Argument is incorrect: Expected `_TypedDictSchema`, found `dict[Unknown | str, Unknown | <class 'str'>]`"
```

Comment:
I'm not sure how we should display `TypedDictSchema` in diagnostics, given that it is not an accessible type. Ideally we would have a special diagnostic for this specific case, but I believe the `TypedDictSchema` type might still show up in the IDE.